specification/computeschedule/resource-manager/Microsoft.ComputeSchedule/preview/2024-08-15-preview/computeschedule.json

```diff
@@ -71,10 +71,10 @@
           }
         },
         "x-ms-examples": {
-          "Operations_ListMax - generated by [MaximumSet] rule - generated by [MaximumSet] rule": {
+          "Operations_ListMax - generated by [MaximumSet] rule - generated by [MaximumSet] rule - generated by [MaximumSet] rule - generated by [MaximumSet] rule - generated by [MaximumSet] rule": {
```

Comment:
this looks peculiar, I think we should fix this.
Similar for all cases as keys of examples.